listenbrainz/webserver/static/css/main.less

```diff
@@ -20,7 +20,8 @@
 @pager-border-radius: @border-radius-base;
 
 body {
-  padding-bottom: 25px;
+  // Leave some space for BrainzPlayer
+  padding-bottom: 60px;
```

Comment:
I believe some math should be explained here in case we want to update the layouts in the future. Something like why 60px? Maybe a variable could help with that.

Reply:
I'll define a variable to make the name clearer.
We can't import the variable from the brainzplayer.less file unfortunately as they will eventually live in different repos